Makefile

```diff
@@ -46,15 +58,57 @@ bats = RESOURCES_DIR=$(RESOURCES_DIR) \
 		--print-output-on-failure \
 		$(1)
 
+define delete-cluster =
+	- k3d cluster delete $(CLUSTER_NAME)
+endef
+
+define create-cluster =
+	$(delete-cluster)
+	k3d cluster create $(CLUSTER_NAME) --wait --timeout $(TIMEOUT) --config $(ROOT_RESOURCES_DIR)/k3d-default.yaml -v /dev/mapper:/dev/mapper
+	$(call kube,  wait --for=condition=Ready nodes --all)
+endef
+
+define install-kubewarden =
+	helm upgrade --install --wait \
+		--kube-context $(CLUSTER_CONTEXT) \
+		--namespace $(NAMESPACE) --create-namespace \
+		$(KUBEWARDEN_CRDS_CHART_RELEASE) $(KUBEWARDEN_CHARTS_LOCATION)/kubewarden-crds
+	helm upgrade --install --wait --namespace $(NAMESPACE) \
+		--kube-context $(CLUSTER_CONTEXT) \
+		--values $(ROOT_RESOURCES_DIR)/default-kubewarden-controller-values.yaml \
+		$(KUBEWARDEN_CONTROLLER_CHART_RELEASE) $(KUBEWARDEN_CHARTS_LOCATION)/kubewarden-controller
+	helm upgrade --install --wait --namespace $(NAMESPACE) \
+		--kube-context $(CLUSTER_CONTEXT) \
+		--values $(ROOT_RESOURCES_DIR)/default-kubewarden-defaults-values.yaml \
+		$(KUBEWARDEN_DEFAULTS_CHART_RELEASE) $(KUBEWARDEN_CHARTS_LOCATION)/kubewarden-defaults
+	$(call kube, wait --for=condition=Ready --namespace $(NAMESPACE) pods --all)
+endef
+
+define install-cert-manager =
+	$(call kube, apply -f https://github.com/jetstack/cert-manager/releases/download/$(CERT_MANAGER_VERSION)/cert-manager.yaml)
+	$(call kube, wait --for=condition=Available deployment --timeout=$(TIMEOUT) -n cert-manager --all)
+endef
+
+define recreate-kubewarden-cluster
+	$(create-cluster)
+	$(install-cert-manager)
+	$(install-kubewarden)
+
+endef
+
+define generate-versioned-resources-dir
```

Comment:
could it be possible to use the newly added `crdVersion` value in the chart? that simplifies things, and makes sure that value is tested.

Reply:
I'm not sure if I understand exactly what you requested. But change the scripts to generated the versioned resources dir to decide which CRD version to use based on the values from the latest `kubewarden-defaults` released.

Reply:
yep, looks good, thanks!